go/install/libnativeinstaller/app.go

```diff
@@ -101,10 +102,45 @@ func UninstallRedirector(runMode libkb.RunMode, log Log) error {
 	return execNativeInstallerWithArg([]string{"--uninstall-redirector"}, runMode, log)
 }
 
+const mountsPresentErrorCode = 7 // See Installer/Installer.m
+
 // InstallFuse calls the installer with --install-fuse.
 func InstallFuse(runMode libkb.RunMode, log Log) error {
 	log.Info("Installing KBFuse")
-	return execNativeInstallerWithArg([]string{"--install-fuse"}, runMode, log)
+	err := execNativeInstallerWithArg([]string{"--install-fuse"}, runMode, log)
+	switch e := err.(type) {
```

Comment:
I think all this code would probably be better in `install_darwin.go` -- that one has all the logic for dealing with error codes, etc.  I'd create a new `InstallFuse()` function, and basically move all this code over there and just call `libnativeinstaller.InstallFuse()` instead of the above `execNative` call.

Reply:
Good call; will do!